docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,133 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
+
+### Why use the Helm Chart?
+- Using this Chart enables you to easily configure many Flink job clusters based on a single template
+- It enables you to use configuration manifests that belong to each other from a central place e.g. ```PodMonitor```
+- You can use helm operations such as ```helm --dry run``` to check for any errors before deployment
+- You can rollback to a previous functioning release with the ```--atomic``` flag
+- Helm includes release versioning
+
+## Prerequisites
+
+
+- Fink Operator Image Version:  ```gcr.io/flink-operator/flink-operator:v1beta1-6``` follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator) to deploy the Operator
+- Flink Image Version: ```flink:1.9.3``` or ```flink:latest```
+- [Helm](https://helm.sh/docs/helm/helm_install/) version 2.x or 3.x
+
+Optional:
+- [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) to use the custom resource ```PodMonitor``` in order to scrape flink-job-cluster metrics
```

Comment:
Can we add a link to this guide https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md#monitoring-with-prometheus?

Reply:
Done!